core/frontend/src/core-frontend.ts

```diff
@@ -75,9 +74,13 @@ export * from "./ViewPose";
 export * from "./ViewRect";
 export * from "./ViewState";
 export * from "./ViewStatus";
+export * from "./extension/Extension";
 export * from "./properties/AngleDescription";
 export * from "./properties/FormattedQuantityDescription";
 export * from "./properties/LengthDescription";
+export * from "./quantity-formatting/QuantityFormatter";
```

Comment:
looks like these 3 lines are a duplicate of lines 52-55. This is the proper place for them so mind removing the ones further up the file?